### PR TITLE
feat(cxl-ui): cxl-dashboard-header add subscription status

### PR DESCRIPTION
**Pull request by pawelkmpt**
https://app.clickup.com/t/86azefd4b

### CODE REVIEW

File: packages/cxl-ui/scss/cxl-dashboard-header.scss [hunk 1]
```diff
@@ -50,6 +50,22 @@
...
+      margin-bottom: var(--lumo-space-s);
+
+      a {
+        color: var(--lumo-primary-color);
+        margin-left: var(--lumo-space-s);
+        text-decoration: none;
+
+        &:hover {
+          text-decoration: underline;
+        }
```
Comment (pawelkmpt) [inline packages/cxl-ui/scss/cxl-dashboard-header.scss:65]: @anoblet @lkraav why do I need to define styles for links inside the component while they're [defined globally](https://conversionxl.github.io/aybolit/?path=/story/cxl-lumo-styles-typography--base)? Would I need to include some global scss? Is it a good idea?

Comment (lkraav) [reply packages/cxl-ui/scss/cxl-dashboard-header.scss:65]: Are these links in Shadow DOM?

Comment (anoblet) [reply packages/cxl-ui/scss/cxl-dashboard-header.scss:65]: I noticed this as well: https://cxlworld.slack.com/archives/C01JABH8AHX/p1704297049011649

You should be able to `@use`, `@import` the typography library: https://github.com/conversionxl/aybolit/blob/c9114aa6564034923007a9b77bd74bf93befdd69/packages/cxl-ui/scss/cxl-certificate-header.scss#L3

Comment (pawelkmpt) [reply packages/cxl-ui/scss/cxl-dashboard-header.scss:65]: I just noticed this style is already being imported. It doesn't have `a` styles though

**Pull request merged**